tekton/src/pipelines/taskdefs/fvt-manage/phase4.yml.j2

```diff
@@ -135,3 +135,44 @@
     - fvt-manage-esig-basic
     - fvt-manage-eaudit-basic
     - fvt-manage-mif-kafka-setup
+
+# Manage FVT BIRT Reports
+- name: fvt-manage-birt-report
```

Comment:
This task needs to be added to phase5 runafter ->https://github.com/ibm-mas/cli/blob/squadci/tekton/src/pipelines/taskdefs/fvt-manage/phase5.yml.j2#L17-L22

Reply:
I made the changes and kept adhoc and birt report tests before the last phase. Is that okay? Let me know if this does not look good. 

Thanks!